package.json

```diff
@@ -18,7 +18,7 @@
     "test": "jest",
     "build": "tsc -p tsconfig.build.json",
     "prepublishOnly": "pnpm build",
-    "dev": "ts-node --env-file=.env.development --env-file .env.local dev",
+    "dev": "ts-node src/index.ts",
```

Comment:
@jleach Do you want to add the config related changes to `dev.ts` as well ? As it helps if we want to run locally using `ngrok`

Reply:
I don't think you need `dev` any more. Just change the `AGENT_ENDPOINTS='https://mediator.example.ca,wss://mediator.example.ca'` in your .env file and `pnpm dev` and you're good to go. To me there shouldn't me any more of a change between dev and test. Thoughts?

Reply:
The dev script connected to ngrok and changed the urls. Now you have to run it manually in another terminal and set the env variables.

It's ok, but does require more setup work

Reply:
@TimoGlastra I don't mind. I can create create a `pnpm dev:standalone` or something for ppl that don't use ngrok. Will create a new ticket for tracking since I merged this one already.